tests/Common/CommonSchemaTest.php

```diff
@@ -405,6 +405,28 @@ public function testGetTableNames(array $pdoAttributes): void
         $db->close();
     }
 
+    /**
+     * @dataProvider \Yiisoft\Db\Tests\Provider\SchemaProvider::pdoAttributes
+     *
+     * @throws NotSupportedException
+     */
+    public function testHasTable(array $pdoAttributes): void
+    {
+        $db = $this->getConnection(true);
+
+        foreach ($pdoAttributes as $name => $value) {
+            $db->getPDO()?->setAttribute($name, $value);
+        }
```

Comment:
```suggestion
    public function testHasTable(array $pdoAttributes): void
    {
        $db = $this->getConnection(true);
```

I'd removed it at all like this